### ISSUE TITLE
Dynamic eye adaptation and objects outside the view

### ISSUE THREAD

**Issue by alex-w** [Stellarium/stellarium#473]
_Original report:_ https://bugs.launchpad.net/stellarium/+bug/967328

At a POV of 66° the moon above the horizon will reduce the apparent brightness of stars and all other objects, reproducing the effect of the moonlight in the reality. But when moving the line of vision so that the moon would be outside the window, this effect will disappear. So the brightness would increase, altough the moon in fact is above the horizon, of course (but only outside the window).

I guess, that is not the desired behaviour of the moonlight-effect, isn't it?

**Comment by gzotti** [comment]
It seems this is very much the intended behaviour. Probably it would make sense to add a special case for the Moon which globally brightens the sky even when it is outside the field of view.

**Comment by axd1967** [comment]
Maybe related to #351 (moonlight)?

**Comment by gzotti** [comment]
Of course. But if you use a narrow FoV e.g. using binoculars or even an empty cardboard tube, or standing in the shadow of a building, you will also see a bit more in real life when the moon does not shine into your eyes. To sum it up:

* A bright moon causes global sky brightening even if not directly visible.
* A bright moon or other object inside the field of view strongly affects dynamic eye adaptation, simulating the direct effect of glare.
